libs/blocks/aside/aside.js

```diff
@@ -46,32 +47,98 @@ function decorateStaticLinks(el) {
   textLinks.forEach((link) => { link.classList.add('static'); });
 }
 
+function decorateMedia(el) {
+  if (!(el.classList.contains('medium') || el.classList.contains('large'))) return;
+  const allMedia = Array.from(el.querySelectorAll('div > p video, div > p picture'));
+  allMedia.some((media) => {
+    const parentP = media.closest('p');
+    const siblingP = parentP?.nextElementSibling;
+    if (!siblingP || siblingP.nodeName !== 'P') return false;
+    const siblingText = siblingP.textContent;
+    const hasFormats = siblingText.match(FORMAT_REGEX)?.index;
```

Comment:
```suggestion
    const hasFormats = FORMAT_REGEX.test(siblingText);
```
Nit: you could optimize this w/ .test()

Reply:
@ryanmparrish I have made the requested changes.